tests/Unit/Core/Webservice/ServerRequirementsCheckerTest.php

```diff
@@ -63,7 +63,10 @@ protected function setUp()
             ->will($this->returnArgument(0));
 
         $this->mockedConfiguration = $this->createMock(Configuration::class);
-        $this->mockedHostingInformation = $this->createMock(HostingInformation::class);
+        $this->mockedHostingInformation = $this->getMockBuilder(HostingInformation::class)
```

Comment:
Why was this change needed?

Reply:
To prevent this error 
![image](https://user-images.githubusercontent.com/1462701/91326403-96ede200-e7c4-11ea-8742-4277bfd88906.png)

For more information: https://phpunit.de/manual/6.5/en/test-doubles.html